docs/reference/objects/product/host.md

```diff
@@ -4,25 +4,29 @@ title: Host
 parent: Product
 ---
 
-When using a `host` object you have access to the following attributes
-Can be accessed on a product like below
+When using a `host` object you have access to the following attributes.<br>
+Can be accessed on a product like below:
```

Comment:
Maybe remove the `<br>Can be accessed on a product like below:` part, as there's no example given.